spec/console/states/game_lost_state_spec.rb

```diff
@@ -1,32 +1,22 @@
 # frozen_string_literal: true
 
 RSpec.describe GameLostState do
-  subject(:lost_state) { described_class.new(console) }
+  subject(:lost_state) { described_class.new(context) }
```

Comment:
You change the name from console to context, but didnt change the logic of the class and the name of it. Context as a part of 'State' pattern should contain just state variable (attr_accessor)